M/mosquitto_client/build_tarballs.jl

```diff
@@ -0,0 +1,39 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "mosquitto_client"
+version = v"2.0.15"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/eclipse/mosquitto.git", "dc75fec606d5c266cff5b3cf2259beee4523cda7")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+export OPENSSL_ROOT_DIR=/workspace/destdir/lib64/
+cd mosquitto/
```

Comment:
```suggestion
cd $WORKSPACE/srcdir/mosquitto/
export OPENSSL_ROOT_DIR=${prefix}/lib64/
```
but I'm not sure OpenSSL is in lib64?  Definitely not for all platforms.

Reply:
Hmm not sure, I had problems with compiling for Windows since it didnt find openssl, with this it compiled successfully for all. But I didnt manually check if the directory exists for all

Reply:
If that's needed only on Windows (and only on 64-bit Windows) then we should use that _only_ in that case, see https://docs.binarybuilder.org/stable/build_tips/#Initiating-different-shell-commands-based-on-target and https://docs.binarybuilder.org/stable/build_tips/#Automatic-environment-variables

Reply:
I updated the file to do as you suggested. One thing I noticed when running the script again, I get

┌ Warning: Minimum instruction set detected for bin/libmosquitto.dll is avx2, not x86_64 as desired.
└ @ BinaryBuilder.Auditor ~/.julia/packages/BinaryBuilder/0CUml/src/Auditor.jl:311
┌ Warning: Minimum instruction set detected for bin/mosquitto_pub.exe is avx2, not x86_64 as desired.
└ @ BinaryBuilder.Auditor ~/.julia/packages/BinaryBuilder/0CUml/src/Auditor.jl:311
┌ Warning: Minimum instruction set detected for bin/mosquitto_rr.exe is avx2, not x86_64 as desired.
└ @ BinaryBuilder.Auditor ~/.julia/packages/BinaryBuilder/0CUml/src/Auditor.jl:311
┌ Warning: Minimum instruction set detected for bin/mosquitto_sub.exe is avx2, not x86_64 as desired.
└ @ BinaryBuilder.Auditor ~/.julia/packages/BinaryBuilder/0CUml/src/Auditor.jl:311

However, I do not know if this is an important thing to fix, or how to fix it.